v0.7/language/bert/evaluate.py

```diff
@@ -0,0 +1,94 @@
+""" Official evaluation script for v1.1 of the SQuAD dataset. """
```

Comment:
@nvzhihanj This is downloaded from https://github.com/allenai/bi-att-flow/blob/master/squad/evaluate-v1.1.py , I think. Could you add the Apache 2.0 license header and add the source where we got this file from? Also, rename this file to `evaluate-v1.1.py`

Reply:
Resolved.